easybuild/easyconfigs/d/datamash/datamash-1.3-foss-2018a.eb

```diff
@@ -0,0 +1,24 @@
+# Author: Piotr Malicki
+
+easyblock = 'ConfigureMake'
+
+name = 'datamash'
+version = '1.3'
+
+homepage = 'https://www.gnu.org/software/datamash/'
+description = "GNU datamash performs basic numeric, textual and statistical operations on input data files"
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+
+source_urls = ['http://ftp.gnu.org/gnu/datamash/']
+sources = ['datamash-%(version)s.tar.gz']
+
+checksums = ['eebb52171a4353aaad01921384098cf54eb96ebfaf99660e017f6d9fc96657a6']
+
+sanity_check_paths = {
+    'files': ['bin/datamash'],
+    'dirs': ['share']
+}
+
+moduleclass = 'data'
+
```

Comment:
@PiotrCM Our automated style check is quite picky about this trailing empty line, please remove?

```
$ eb --check-style --from-pr 6693
== temporary log file in case of crash /Users/kehoste/work/TMP/eb-p87KgB/easybuild-HpxYt0.log

*** WARNING: Using easyconfigs from unstable (pending/failed tests or merge conflict) PR #6693 ***

Running style check on 1 easyconfig(s)...
/Users/kehoste/work/TMP/eb-p87KgB/files_pr6693/d/datamash/datamash-1.3-foss-2018a.eb:24:1: W391 blank line at end of file
[FAIL] /Users/kehoste/work/TMP/eb-p87KgB/files_pr6693/d/datamash/datamash-1.3-foss-2018a.eb
ERROR: One or more style checks FAILED!
```